src/components/faceVerification/standalone/App.jsx

```diff
@@ -39,6 +39,7 @@ const App = () => {
             apiKey={Config.posthogApiKey}
             options={{ host: Config.posthogHost, sendFeatureFlagEvent: false }}
             autocapture={false}
+            person_profiles="identified_only"
```

Comment:
whats this?

Reply:
removed, was used for testing posthog events